### ISSUE TITLE
the color of output is error

### ISSUE THREAD

**Issue by rk764457656** [AndreiiArhire/Learned-Lightweight-Smartphone-ISP-with-Unpaired-Data#2]
I try the model on t [Fujifilm UltraISP] datasets with your pre-trianed model.  The output of the model looks strange. 

<img width="274" height="275" alt="Image" src="https://github.com/user-attachments/assets/891511e6-7fa1-4632-8635-3764149dda20" />

**Comment by AndreiiArhire** [comment]
Hi,

Please use the ```pretrained/run_inference.py``` script to run the pre-trained models.
I have run all nine pre-trained models on the Fujifilm dataset using your sample image (0.png from training) and obtained the expected results. The image you have uploaded does not match the expected behavior.

In the Fujifilm dataset, the patches are only 256×256. While the overall color reproduction is reasonable, the small patch size limits the color accuracy.
In contrast, in the ZRR dataset the patches are 448×448, and the color is better captured, as demonstrated in the examples from the paper.

**Comment by rk764457656** [comment]
Hi @AndreiiArhire 
Thank you for your reply. Through the method you described, the problem has been solved. The reason for the problem lies in the fact that I used the model that path is ./checkpoint/pretrained_20000.pth
